tests/subcommands.rs

```diff
@@ -0,0 +1,59 @@
+use assert_cmd::prelude::*;
+use predicates::str::contains;
+
+mod common;
+
+use common::MaskCommandExt;
+
+#[test]
+fn positional_arguments() {
+    let (_temp, maskfile_path) = common::maskfile(
+        r#"
+
+## services
+
+> Commands related to starting, stopping, and restarting services
+
+### services start (service_name)
+
+> Start a service.
+
+~~~bash
+echo "Starting service $service_name"
+~~~
+
+### services stop (service_name)
+
+> Stop a service.
+
+~~~bash
+echo "Stopping service $service_name"
+~~~
+
+#### services stop (service_name) now
+
+> Immediately stop without graceful shutdown
+
+~~~bash
+echo "Stopping $service_name now"
+~~~
+"#,
+    );
+
+    common::run_mask(&maskfile_path)
+        .cli("services start my_fancy_service")
+        .assert()
+        .stdout(contains("Starting service my_fancy_service"))
+        .success();
+
+    /*
+     * Doesn't really works.
+     * It triggers the 'service stop $name
+     *
```

Comment:
Yupp, i think this makes sense. For now I don't see a real need to support command names _after_ required/optional args. The workaround is just to make `now` another required arg like `(when)` for example.

I feel like we can probably delete this commented out block and also delete the ` #### services stop (service_name) now` section from the maskfile above 👍

Reply:
👍  I'll remove it from here and the `README.md`, which is where I got it from.